test/webgl/index.js

```diff
@@ -1,24 +1,27 @@
+// No headless
+// import './context-no-headless.spec';
```

Comment:
Good question
No that file is imported from a different test run (without headless-gl imported).
We run with and without headless gl. I have pushed a commit that removes the commented line in this index.js file.